src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractRoslynTableDataSource.cs

```diff
@@ -16,6 +17,17 @@ public AbstractRoslynTableDataSource(Workspace workspace) : base(workspace)
             ConnectToSolutionCrawlerService(workspace);
         }
 
+        protected ImmutableArray<DocumentId> GetDocumentGroupKey(Solution solution, DocumentId documentId)
```

Comment:
This is extremely oddly named.  It seems to be saying GetDocumentsWithSameFilePath.